chcemvediet/apps/inforequests/templates/inforequests/index/index.html

```diff
@@ -0,0 +1,47 @@
+{# vim: set filetype=htmldjango shiftwidth=2 :#}
+{% extends "main/base/single_column.html" %}
+{% load trans from i18n %}
+{% load range from poleno.utils %}
+
+{% comment %}
+ %
+ % Context:
+ %  -- inforequest_page: django.core.paginator.Paginator(chcemvediet.apps.inforequests.models.Inforequest.objects)
+ %
+{% endcomment %}
+
+
+{% block title %}{% trans 'inforequests:index:title' %} | {{ block.super }}{% endblock %}
+
+{% block content %}
+  <h1>{% trans 'inforequests:index:heading' %}</h1>
+  {% include "inforequests/index/texts/index.html" %}
+  {% include "inforequests/index/published.html" with published_inforequests=inforequest_page.object_list %}
+  <nav class="text-center">
+    <ul class="pagination">
+      {% if inforequest_page.number > 1 %}
+        <li><a href="?page=1">&laquo;</a></li>
```

Comment:
Tento paging je identicky ako v `obligees` app vsak? V takom pripade je ho lepsie vydelit do vseobecneho subtemplatu a ten reusnut na oboch miestach. Kedze je to subtemplate spolocny pre viacero app, treba ho dat do spolocnych templates napr `chcemvediet/templates/main/snippets/paging.html`

Jediny rozdiel oproti `obligees` app je rozny nazov premennej. To vyriesime tak, ze template includneme s
`{% include ... with page=inforequest_page %}`

Reply:
Ano paging je rovnaky ako v `obligees`.